src/katsdpcontroller/fake_servers.py

```diff
@@ -157,8 +157,18 @@ def __init__(self, *args, **kwargs) -> None:
             output: [np.full((1,), self.DEFAULT_GAIN, np.complex64) for _ in range(self.N_POLS)]
             for output in self._output_names
         }
-        for pol in range(self.N_POLS):
-            for output in self._output_names:
+
+        for output in self._output_names:
+            self.sensors.add(
+                Sensor(
+                    str,
+                    f"{output}.dither-seed",
+                    "Random seed used in dithering for quantisation",
+                    default=f"{2**64 - 1}",
```

Comment:
Maybe add a comment that you're using the largest value that katgpucbf could return here, since that's the most likely value to break anything.